Makefile

```
@@ -26,7 +26,7 @@ STYLE_CHECK_DIRS := src demos/common/cpp/src demos/image_classification/cpp demo
 HTTP_PROXY := "$(http_proxy)"
 HTTPS_PROXY := "$(https_proxy)"
 NO_PROXY := "$(no_proxy)"
-JOBS ?= $(shell python3 -c 'import multiprocessing as mp; print(mp.cpu_count())')
+JOBS ?= $(shell python3 -c 'import multiprocessing as mp; print(int(mp.cpu_count()/2))')
```

Alternative approach using pure bash
```suggestion
CORES_PER_SOCKET=$(lscpu | awk '/^Core\(s\) per socket:/ {print $NF}')
SOCKETS=$(lscpu | awk '/^Socket\(s\):/ {print $NF}')
CORES_TOTAL=$(( SOCKETS * CORES_PER_SOCKET ))
JOBS ?=$(CORES_TOTAL)
```